src/css/profile/mobile/common/popup.less

```diff
@@ -163,6 +187,28 @@
 		flex-grow: 0;
 		width: 100%;
 
+		&::before {
+			content: "";
+			position: absolute;
+			left: 24 * @px_base;
+			width: calc(~"100%" - 48 * @px_base);
+			height: 0.75 * @px_base;
+			stroke-width: 0.5;
+			visibility: hidden;
+			background-color: var(--popup-scroll-divider-color);
+
+			@media (orientation: portrait) {
+				bottom: 80 * @px_base;
+			}
+			@media (orientation: landscape) {
+				bottom: 50 * @px_base;
+			}
+		}
+
+		&.bottomDivider::before {
+			visibility: visible;
```

Comment:
Element hidden by visible still hold a place. Is this intended?

Reply:
That's right. 
The only place in the dialog where scrolls are generated is in cases where there are many list items.
I intentionally applied the visibility option.